packages/protocol/test/governance/governance.ts

```diff
@@ -623,15 +773,18 @@ contract('Governance', (accounts: string[]) => {
             proposalId: new BigNumber(1),
             proposer: accounts[0],
             deposit: new BigNumber(minDeposit),
-            timestamp: new BigNumber((await web3.eth.getBlock('latest')).timestamp),
+            timestamp: matchAny,
```

Comment:
Why isn't the timestamp == afterTimestamp? Ganache will only mine blocks when there are txs to mine

Reply:
Still have this question